caching-service/src/main/resources/application.yml

```diff
@@ -138,6 +141,7 @@ spring:
 logging:
     level:
         ROOT: INFO
+        org.jgroups: TRACE
```

Comment:
Wouldn't `INFO` be enough?

Reply:
INFO level is not enough to debug issues around jgroups connections in case they arise.